critiquebrainz/db/users_test.py

```diff
@@ -0,0 +1,18 @@
+from sqlalchemy.exc import DatabaseError
+from critiquebrainz.data.testing import DataTestCase
+from critiquebrainz.data.model.user import User
+from critiquebrainz.db import users
+
+
+class UserTestCase(DataTestCase):
+    def setUp(self):
+        super(UserTestCase, self).setUp()
+        self.user1 = User.get_or_create("test", musicbrainz_id='0')
+        self.user2 = User.get_or_create("test1", musicbrainz_id='1')
+        self.admins = [self.user1.display_name, self.user2.display_name]
+
+    def test_response_type(self):
```

Comment:
Typically in a test case each method tests a particular set of features in a given function. The way to name them is the following: `test_get_many_by_mb_username`. Can also append clarification of particular feature the test is for, but isn't necessary if there's just one test for a function.